src/hook/useCategory.ts

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { CategoryOption } from '../types/common';
+
+const useCategory = () => {
+  const [category, setCategory] = useState<CategoryOption>('전체');
+
+  const selectCategory = (category: string) => {
+    setCategory(category as CategoryOption);
+  };
```

Comment:
```suggestion
  const selectCategory = (category: string) => {
    setCategory(category);
  };
```

이렇게 prop에 타입을 지정해주면 되지 않을까요!?

Reply:
```ts
const selectCategory = (category: CategoryOption) => {
    setCategory(category);
  };
```

이렇게 적용하면 될까요??